packages/bun-types/bun.d.ts

```diff
@@ -6421,6 +6421,66 @@ declare module "bun" {
      */
     timestamp?: number | Date,
   ): Buffer;
+
+  /**
+   * Types for `bun.lock`
+   */
+  type BunLockFile = {
+    lockfileVersion: 0;
+    workspaces: {
+      [workspace: string]: {
```

Comment:
Consider making this a separate type so others can import it in their own code

Reply:
good idea, i just wasn't sure on naming and `BunLockFIle["workspaces"]["anything"]` type thing is something i have used too much sadly.

I'll try to make the types here more abstracted and reusable

Reply:
done